homeassistant/components/husqvarna_automower/select.py

```diff
@@ -0,0 +1,66 @@
+"""Creates a select entity for the headlight of the mower."""
+
+import logging
+
+from aioautomower.exceptions import ApiException
+from aioautomower.model import HeadlightModes
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AutomowerDataUpdateCoordinator
+from .entity import AutomowerControlEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+HEADLIGHT_MODES: list = [option.lower() for option in list(HeadlightModes)]
```

Comment:
No i mean literally


```suggestion
HEADLIGHT_MODES: list = ["a", "b"]
```

This way if the source enum changes the code doesn't immediately break the integration

Reply:
I've changed it now, but not completely literally. Because Martin once said "We shouldn't need to handle magic api values directly as a user of the library." So I try to avoid this.